.circleci/config.yml

```diff
@@ -8,9 +8,18 @@ jobs:
       - checkout
       - run: npm ci
       - run: npm t
+  test_in_browser:
+    docker:
+      - image: circleci/node:16
+    steps:
+      - checkout
+      - run: sudo apt-get install xvfb
+      - run: npm ci
```

Comment:
I think you need to run `npm run build` here first? I notice in the [one I made](https://github.com/TehShrike/abstract-state-router/pull/140/files) that it runs `npm run test` which has `build` as a side effect.

Reply:
The browser script runs it too... https://github.com/TehShrike/abstract-state-router/blob/master/package.json#L9 ...I'm not sure what's going wrong :-x

Reply:
You've got the browser scripts in a different job. My understanding is that assets weren't available across jobs unless you specified it or something?